main.go

```diff
@@ -131,6 +134,20 @@ func updateConfig(configPath, backupConfig string) (config *Config, err error) {
 	return config, err
 }
 
+type IPInfo struct {
+	Country string `json:"country"`
+}
+
+func openBrowser(url string, l *logs.Logger) {
+	switch runtime.GOOS {
+	case "windows":
+		exec.Command("rundll32", "url.dll,FileProtocolHandler", url).Start()
```

Comment:
(Optional) Please explicitly check for error or explicitly ignore it with
```suggestion
		_ = exec.Command("rundll32", "url.dll,FileProtocolHandler", url).Start()
```